app/shared/src/commonMain/kotlin/ui/subject/episode/mediaFetch/MediaSelectorItem.kt

```diff
@@ -187,9 +189,13 @@ fun MediaSelectorItemLayout(
             containerColor = if (selected) {
                 MaterialTheme.colorScheme.secondaryContainer
             } else {
-                MaterialTheme.colorScheme.surfaceContainerHighest
+                Color.Transparent
             },
         ),
+        border = if (!selected) BorderStroke(
+            width = 1.dp,
+            color = MaterialTheme.colorScheme.outline.weaken(),
```

Comment:
```suggestion
            color = MaterialTheme.colorScheme.outlineVariant,
```

用 outlineVariant 更合适